src/Env/Config/Manala.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Env\Config;
+
+/**
+ * Make.
```

Comment:
@chalasr : No no no. I mean... unless `make` has something to do with the `Manala` class on which this docblock is... :trollface: